src/dir.targets

```diff
@@ -157,9 +154,39 @@
     <DebugEngines>{2E36F1D4-B23C-435D-AB41-18E608940038}</DebugEngines>
   </PropertyGroup>
 
+  <Target Name="CheckTestCategories"
+          AfterTargets="Test"
+          Condition="'$(IsTestProject)'=='True' And '$(SkipTests)'!='True'">
+    <PropertyGroup>
+      <!-- By default project has inner- and outer- loop tests. -->
+      <TestCategories Condition="'$(TestCategories)'==''">InnerLoop;OuterLoop</TestCategories>
```

Comment:
I feel like this PR was a bad idea from the start. Now when someone creates OuterLoop tests in a project they would also need to know that you have to change a csproj and add a category. I don't expect people to remember that as I would probably forget about this after few months even if I was the one who implemented it. I would rather wait until new version of Xunit (they already fixed they bug) and keep only traits. I find that having separate test project categories and test categories is an unnecessary effort which leads to eventually saving few seconds during the build. I'd rather wait that time and not have to remember to add anything in your csproj.